modules/sdk-coin-sol/test/unit/sol.ts

```diff
@@ -1509,6 +1509,80 @@ describe('SOL:', function () {
       sandBox.restore();
     });
 
+    it('should take OVC output and generate a signed sweep transaction', async function () {
```

Comment:
nit: tests for non-happy paths

Reply:
yeah we have tested e2e with sol, and it works